manuscript/applications.tex

```diff
@@ -0,0 +1,36 @@
+\section{Other Uses of Dynamic Deployment}
+The concept of dynamic demand-driven deployment is not new,
+and has been used in various applications. Various algorithms
+are used to optimize deployment of facility or utilization
+of given sources.
+
+\subsection{Aircraft Dispatch}
```

Comment:
Oh this should be a different Pull request. I'm sorry I thought I created a separate branch.